### PR TITLE
Sync up CHANGELOG to the v0.1.0-beta.1 milestone

### PR DESCRIPTION
**Pull request by benashz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
 ## Unreleased
 
 Bugs:
-* Helm Chart: fix deployment templating so setting `controller.kubernetesClusterDomain` works as defined in values.yaml. [GH-183](https://github.com/hashicorp/vault-secrets-operator/pull/183)
+* Helm Chart: fix deployment templating so setting `controller.kubernetesClusterDomain` works as defined in values.yaml: [GH-183](https://github.com/hashicorp/vault-secrets-operator/pull/183)
+* VaultPKISecret (VPS): Ensure `Spec.AltNames`, and `Spec.IPSans`are properly formatted for the Vault request: [GH-130](https://github.com/hashicorp/vault-secrets-operator/pull/130)
+* VaultPKISecret (VPS): Make `Spec.OtherSANS` a string slice (**breaking change**): [GH-190](https://github.com/hashicorp/vault-secrets-operator/pull/190)
```
Comment (tvoran) [inline CHANGELOG.md:6]: I could also see this going into a separate `Changes` section, to make it more obvious when there's a breaking change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 
 Features:
-* VaultDynamicSecrets (VDS): CRD is extended with `Revoke` field which will result in the dynamic secret lease being revoked on rotation and CR deletion. Note: The VaultAuthMethod referenced by the VDS Secret must have a policy which provides `["update"]` on `sys/leases/revoke`. [GH-143](https://github.com/hashicorp/vault-secrets-operator/pull/143)
-* VaultAuth: Adds support for the JWT authentication method which either uses the JWT token from the provided secret reference, or a service account JWT token that VSO will generate using the provided service account. [GH-131](https://github.com/hashicorp/vault-secrets-operator/pull/131)
-* VaultDynamicSecrets (VDS): New `RenewalPercent` field to control when a lease is renewed [GH-170](https://github.com/hashicorp/vault-secrets-operator/pull/170)
+* VaultDynamicSecrets (VDS): CRD is extended with `Revoke` field which will result in the dynamic secret lease being revoked on rotation and CR deletion. Note:
+  The VaultAuthMethod referenced by the VDS Secret must have a policy which provides `["update"]` on `sys/leases/revoke`: [GH-143](https://github.com/hashicorp/vault-secrets-operator/pull/143)
```
Comment (tvoran) [inline CHANGELOG.md:10]: Perhaps we don't need the acronym (VPS, VDS, etc.) when they're not used in the summary?

**Review by tvoran — APPROVED** [review]

**Review by kschoche — APPROVED** [review]
Looks great, thanks for taking this on!

File: CHANGELOG.md (hunk 1 above)
Comment (benashz) [reply CHANGELOG.md:6]: Added in https://github.com/hashicorp/vault-secrets-operator/pull/193/commits/c16a3dd06597cda45ce3f1fdf09dd6e3ab3c48f5

**Pull request merged**